IPCheck/dashAnalysis.sh

```diff
@@ -1,118 +1,120 @@
 #!/bin/bash
+#*******************************************************************************
+# Copyright (c) 2021 The Eclipse Foundation
 #
-# ********************************************************************************
-# * Copyright (c) 2021, The Eclipse Foundation
-# *
-# * This program and the accompanying materials are made available under the
-# * terms of the Eclipse Public License 2.0 which is available at
-# * http://www.eclipse.org/legal/epl-2.0.
-# *
-# * SPDX-License-Identifier: EPL-2.0
-# *
-# * Contributors:
-# *   phkrief - initial implementation
-# ********************************************************************************
+# This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     phkrief - initial API and implementation
+#*******************************************************************************
+
 #
 #--------------------------------------------------------------
 # Runs the Eclipse DASH dependency analysis on a (Maven, Yarn or Nodes) repository 
 #		$1 	: To select between "Maven" | "Nodes" | "Yarn"
-#		$2	: Path of the folder containing the file to parse. The repo is supposed to be located under the $GIT_REPOS folder
-#		$3	: Optional - If the path of the folder is not the name of the repo then this third argument can specify the name of the repo
+#		$2	: The Path to the repository/folder where the config file is located
+#		$3	: Name to use (usually the name of the repo) 
```

Comment:
nit: This documentation for the 3rd argument does not really fit anymore. Maybe replace with the usage info printed for `<destination>` in line 32.